packages/core/src/internal/clients/pubsub/AbstractTopicClient.ts

```diff
@@ -46,6 +54,56 @@ export abstract class AbstractTopicClient implements ITopicClient {
     topicName: string,
     options: SubscribeCallOptions
   ): Promise<TopicSubscribe.Response> {
-    return await this.client.subscribe(cacheName, topicName, options);
+    return await this.pubsubClient.subscribe(cacheName, topicName, options);
+  }
+
+  /**
+   * Deletes a webhook
+   *
+   * @param {WebhookId} id - The webhook id to be deleted
+   * @returns {Promise<DeleteWebhook.Response>} -
+   * {@link DeleteWebhook.Success} on success.
+   * {@link DeleteWebhook.Error} on failure.
+   */
+  public async deleteWebhook(id: WebhookId): Promise<DeleteWebhook.Response> {
```

Comment:
food for thought comment here. I'm trying to focus extra attention on the public APIs in this PR since this is the first SDK where we will be exposing them.

When I first saw what you did for `WebHookId` I thought it was nice to encapsulate that into a single object. But then looking over it further I'm wondering if it would be more consistent with the rest of our SDK APIs to just keep it as two args: cacheName, webhookName.

e.g., with the various Topic APIs, we don't encapsulate a `TopicId`, we just expose both things as args. Kinda true with collections too.

And, like, in the CacheClient, we made this `.cache()` interface that provides all of the CacheClient APIs with the cache name curried. If we wanted to do that in the future for TopicClient as well, these signatures would be a little out of place.

Open to discussion on this, I don't have an overwhelmingly strong opinion about it, but it is feeling to me like it would be a bit more consistent to break the two things into separate args.